docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"following"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
```

Comment:
following->resolved is done. Regarding bluebird instances: each application probably uses single bluebird instance, different instances come from 3rd party dependencies, right? Why would somebody like to subscribe to these events that happen outside of his own code?
If you want it on (window|process) anyway, should it be something like (window|process).Promise.on()?